src/components/onboarding.tsx

```diff
@@ -125,13 +117,6 @@ export default function Onboarding() {
 		[ setFileForImport ]
 	);
 
-	useIpcListener( 'add-site', () => {
-		if ( isAnySiteProcessing || ! needsOnboarding ) {
-			return;
-		}
-		onAddSite();
-	} );
```

Comment:
I assume that we no longer need this because the `Add site` menu item will be disabled. Is this accurate?

Reply:
Yeah we initially added in https://github.com/Automattic/studio/pull/492/files when we were trying to make that menu item submit the form to add a site instead of being disabled when the user needs onboarding but it is no longer necessary (as we decided to remove this behavior).